opencti-platform/opencti-front/src/private/Root.tsx

```diff
@@ -277,6 +279,7 @@ const RootComponent: FunctionComponent<RootComponentProps> = ({ queryRef }) => {
   const settings = useFragment<RootSettings$key>(rootSettingsFragment, settingsFragment);
   const me = useFragment<RootMe_data$key>(meUserFragment, meFragment);
 
+  const entitySettingsData = entitySettings?.edges?.map((setting) => (useFragment<EntitySettingSettings_entitySetting$key>(entitySettingFragment, setting.node)));
```

Comment:
As a start, may be we don't need fragment here to simplify

Reply:
What do you have in mind ?
I chose this like the previous lines above.

Reply:
To get the data directly in the query